src/Administration/Resources/app/administration/src/module/sw-settings-rule/view/sw-settings-rule-detail-base/index.js

```diff
@@ -87,12 +96,45 @@ export default {
     methods: {
         createdComponent() {
```

Comment:
maybe you could run the async calls in parallel, since they don’t seem related?

```suggestion
        async createdComponent() {
            await Promise.all([
                this.loadCustomFieldSets(),
                this.loadProductStreamIndexingConfig(),
            ]);
        }
```